contracts/ethregistrar/LinearPremiumPriceOracle.sol

```diff
@@ -1,40 +1,46 @@
 pragma solidity >=0.8.4;
```

Comment:
Assuming it still compiles, I think we can just revert all the changes to this file, since we're not going to deploy the modified contract.